packages/config/adders/official.ts

```diff
@@ -0,0 +1,10 @@
+import type { AdderCategories } from "../categories";
+
+export const adderCategories: AdderCategories = {
+    css: ["tailwindcss", "bulma", "bootstrap"],
+    db: ["drizzle"],
+    markdown: ["mdsvex"],
+    tools: ["storybook", "routify"],
```

Comment:
I wonder about making this something like `router` and `uiDevelopment` as tools is a bit generic and could become a catch-all

Reply:
I couldn't imagine any other adders getting into those categories at this point. We can always change that later!